### DISCUSSION TITLE
How do I play a custom map?

### DISCUSSION THREAD

**Discussion by MadeInDutch** [CorsixTH/CorsixTH#2141] [General Diagnostics]
I reckon this question has been asked a couple of times already but I couldn't really find a conclusive answer to it.

I've spent a couple of hours working on a custom map and I'd love to play it as a single scenario. But I don't know how to. I tried moving the _.map_ file to the campaigns folder but that didn't work. I'm kinda stumped.

What do I do?

**Comment by lewri** [comment]
You'll need to also attach a .level file to it. It is the level file that appears in the custom game menu
https://github.com/CorsixTH/CorsixTH/wiki/Custom-Levels#creating-custom-levels

**Reply by MadeInDutch** [reply]
Custom game menu? I don't understand what the article means.

**Reply by lewri** [reply]
Sorry, I meant "single scenario".

**Reply by lewri** [reply]
I had been away from a PC...
Download and unzip the below zip file which has a `.level` file in. Put it with the `.map` file into the "Levels" folder in `C:\<user>\Appdata\Roaming\CorsixTH\` (no admin rights needed) or `C:\Program Files\CorsixTH` (may need admin rights)
Open the `.level` file in a text editor and change the `%Name = "LEVEL NAME HERE"` to the name you want to give the level/map.
Also change the `%MapFile = "MAP_FILENAME.map"` to the filename of your map.

Save the .level file and launch CorsixTH -- It should appear in the Single Scenario menu
[MadeInDutch.zip](https://github.com/CorsixTH/CorsixTH/files/8550522/MadeInDutch.zip)

NB: That is just a copy of the example.level file, so it should provide enough configuration to play the map.

**Reply by MadeInDutch** [reply]
It works, thank you!

**Reply by TheCycoONE** [reply]
If anything is unclear in the wiki that this answer helped with, please take the time to update.

**Reply by lewri** [reply]
I think it's actually devoid of providing this information clearly. Not good.
